src/test/java/kitchenpos/MenuTest.java

```diff
@@ -0,0 +1,214 @@
+                    () -> assertThat(sut.getPrice()).isEqualTo(menuRequest.getPrice())
+            );
+
+        }
+
+        @DisplayName(value = "메뉴의 가격은 0원 이상이어야 한다.")
+        @Test
+        void invalidMenuAmount() {
+            //상품명이나 가격이 없을때 에러처리
+            Menu menu = createMenu(후라이드치킨_MINUS_PRICE);
+            assertThatIllegalArgumentException().isThrownBy(() -> menuService.create(menu));
+        }
+
+        @DisplayName(value = "메뉴의 주문 상품이 1개 이상 존재해야 합니다")
+        @Test
+        void invalidMenuProduct() {
+            MenuGroup menuGroup = createMenuGroup(한마리메뉴_MENU_GROUP_NAME, 후라이드치킨_MENU_GROUP_UUID);
+            menuGroupRepository.save(menuGroup);
+            Menu menu = createMenu(후라이드치킨_MENU_UUID, 후라이드치킨_MENU_NAME, 후라이드치킨_DEFAULT_PRICE, menuGroup, 후라이드치킨_MENU_GROUP_UUID);
+            assertThatIllegalArgumentException().isThrownBy(() -> menuService.create(menu));
+        }
+
+        @DisplayName(value = "메뉴 내부 상품의 수량은 0개 이상이어야 합니다.")
+        @Test
+        void menuProductNotEqualsProductSize() {
+            Product product = createProduct();
+            MenuGroup menuGroup = createMenuGroup();
+            menuGroupRepository.save(menuGroup);
+            var menuProducts = createMenuProduct(product, MINUS_QUANTITY, 후라이드치킨_PRODUCT_UUID);
+            Menu menu = createMenu(menuGroup, menuProducts);
+            assertThatIllegalArgumentException().isThrownBy(() -> menuService.create(menu));
+        }
+
+        @DisplayName(value = "등록하려는 메뉴의 가격이 메뉴에 포함된 상품의 총 가격보다 높으면 안됩니다.")
+        @Test
+        void invalidTotalMenuAmount() {
+            menuGroupRepository.save(createMenuGroup());
+            Menu menu = createMenu(후라이드치킨_OVER_PRICE);
+
+            assertThatIllegalArgumentException().isThrownBy(() -> menuService.create(menu));
+        }
+
+        @DisplayName(value = "메뉴의 이름이 없거나 비속어가 들어가 있으면 안됩니다.")
+        @Test
+        void invalidMenuName() {
+            Product product = createProduct(MenuTest.후라이드치킨_PRODUCT_UUID, PRODUCT_NAME_비속어, 후라이드치킨_DEFAULT_PRICE);
+            MenuGroup menuGroup = createMenuGroup(한마리메뉴_MENU_GROUP_NAME, 후라이드치킨_MENU_GROUP_UUID);
+            menuGroupRepository.save(menuGroup);
+            List<MenuProduct> menuProducts = List.of(createMenuProduct(product, 1, 후라이드치킨_PRODUCT_UUID));
+            Menu menu = createMenu(후라이드치킨_MENU_UUID, 후라이드치킨_PROFANITY_MENU_NAME, 후라이드치킨_DEFAULT_PRICE, menuGroup, menuProducts, 후라이드치킨_MENU_GROUP_UUID);
+
+            assertThatIllegalArgumentException().isThrownBy(() -> menuService.create(menu));
+        }
+    }
+
+
+
+    @DisplayName(value = "메뉴 가격 수정 기능")
+    @Nested
+    class MenuChangePriceTest {
+        private static final BigDecimal 후라이드치킨_MINUS_PRICE = new BigDecimal(-10);
+        private static final BigDecimal 후라이드치킨_CHANGE_PRICE = new BigDecimal(19000);
+
+        @DisplayName(value = "메뉴가격을 수정할 수 있다.")
+        @Test
+        void changeMenu() {
+            menuRepository.save( createMenu());
```

Comment:
테스트 코드도 리팩터링 대상이기 때문에 코드 정렬은 항상 수행해주세요 : ) 

```suggestion
            menuRepository.save(createMenu());
```

Reply:
정렬을 안해놨네요ㅠ 수정 완료했습니다